azdev/utilities/const.py

```diff
@@ -6,10 +6,36 @@
 
 import sys
 
+DEFAULT_RESULT_FILE = 'test_results.xml'
 COMMAND_MODULE_PREFIX = 'azure-cli-'
 EXTENSION_PREFIX = 'azext_'
 EXT_REPO_NAME = 'azure-cli-extensions'
 IS_WINDOWS = sys.platform.lower() in ['windows', 'win32']
+CONFIG_NAME = 'config'
+ACTIVATE_PS = 'Activate.ps1'
+PS1_VENV_SET = '$env:VIRTUAL_ENV'
+SCRIPTS = 'Scripts'
+VIRTUAL_ENV = 'VIRTUAL_ENV'
+VENV_CMD = 'python -m venv --system-site-packages '
+VENV_CMD3 = 'python3 -m venv --system-site-packages '
+AZ_CONFIG_DIR = 'AZURE_CONFIG_DIR'
+AZ_AZDEV_DIR = 'AZDEV_CONFIG_DIR'
+AZ_DEV_SRC = 'dev_sources'
+AZ_CLOUD = 'AzureCloud'
+CLOUD_SECTION = 'cloud'
+EXT_SECTION = 'extension'
+CLI_SECTION = 'clipath'
+EVN_AZ_CONFIG = '$env:AZURE_CONFIG_DIR'
+EVN_AZ_DEV_CONFIG = '$env:AZDEV_CONFIG_DIR'
+PIP_E_CMD = 'pip install -e '
```

Comment:
Could we leverage `pip_cmd`?

Reply:
Not in the scenario this is use in. It is used in the venv class and it is used to build a larger string (representing a shell command) depending on windows or unix.

Reply:
I found many places that you could leverage the code in `azdev.utilites.command`, leaving those would make code look like redudant and messy.

Reply:
Will look at this when we fixing the subprocess.cmd function.